apps/use-after-free/null-pointer.txt

```diff
@@ -0,0 +1,16 @@
+Exception in thread "main" java.lang.NullPointerException
```

Comment:
What is this file and sigsev.txt?

Reply:
These two files show two possible errors that occur when running this program. I'll include a readme/more documentation with some comments on what these mean. More or less, I put them here as "proof" that the use after free not only causes an error, but has the potential to cause multiple different errors.